src/style/utils/at_opacity.js

```diff
@@ -0,0 +1,9 @@
+export default (base) => {
+  const parse = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(base);
```

Comment:
this expression could be shortened to:
```(/[^#]{2}/g)```
```match``` could be used to split that color into an array with three hex digit pairs